terraform-modules/k8s/k8s-cluster.tf

```diff
@@ -59,36 +55,37 @@ resource "google_container_cluster" "cluster" {
   # CIS compliance: Enable Alias IP Ranges. According to the terrafform
   # docs, setting these values blank gets default-size ranges automatically
   # chosen: https://www.terraform.io/docs/providers/google/r/container_cluster.html#ip_allocation_policy
-  ip_allocation_policy = ["${var.ip_allocation_policy}"]
+  ip_allocation_policy = var.ip_allocation_policy
```

Comment:
"Fun" TF 0.12 detail: configuration that used to be specified by repeated nested blocks can _sometimes_ be replaced by direct assignment of an array of objects, but only if the provider opts into that behavior for the specific field.

Reply:
obviously